birdhouse/components/cowbird/user_workspace_readme.ipynb

```diff
@@ -0,0 +1,130 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6eb979d5-fc6f-456b-9931-6209958f4bae",
+   "metadata": {},
+   "source": [
+    "# README\n",
+    "\n",
+    "This is the user workspace for the JupyterLab environment.\n",
+    "\n",
+    "## Home Directory Layout\n",
+    "\n",
+    "```\n",
+    "notebook_dir\n",
+    "├── README.ipynb\n",
+    "├── public\n",
+    "├── tutorial-notebooks\n",
+    "└── writable-workspace\n",
+    "    ├── notebooks\n",
+    "    ├── shapefile_datastore\n",
+    "    └── wps_outputs \n",
```

Comment:
I am a little bit surprised the `wps_outputs/` is under `writetable-workspace/`.  Is this only the user's output or other output shared to the user?  If also other output shared to the user, doesn't it imply it is writable while it is not?

And does the user really have write access to their own wps output since the actual author is the WPS service?

Reply:
The `wps_outputs` directory under the `writable-workspace` only contains the own user's WPS outputs files. 
Any public `wps_outputs` files will be found under `/notebook_dir/public/...`.

The `wps_outputs` directory here is indeed read-only. I guess this can be quite confusing to have it under `writable-workspace`, but it seemed the easiest choice considering the different design restrictions with Cowbird and birdhouse.
Cowbird has to create hardlinks for user WPS outputs to a directory that is specific to the related user, which would logically be the user's workspace, that is mounted as `writable-workspace` here.

I guess the user's WPS outputs hardlinks would have to be stored in a new separated distinct directory, outside of the user_workspaces, so that JupyterHub could mount it outside of the `writable-workspace`? This would require some changes in Cowbird and birdhouse though.

Another option (when Cowbird is activated) would be not to use the `writable-workspace` name, but something else, such as `user-workspace`, but I'm not sure if we want to touch that dir name.

Reply:
Given that the real nature of `writable-workspace` is to be the `user-workspace`, I would lean toward the rename solution.
The important notion to highlight with this directory is that it is the location the user should use to persist data across sessions, as mentioned in another comment. However, that does not mean this data necessarily has to be writable or managed directly by the user.

Rename does not have to be done in this PR either. Probably safer to have it separately to announce it as a change by itself, since it could impact some users' scripts.